src/Tasks/AssemblyDependency/AssemblyInformation.cs

```diff
@@ -567,30 +567,20 @@ internal static string GetRuntimeVersion(string path)
 #if FEATURE_MSCOREE
             if (NativeMethodsShared.IsWindows)
             {
-                StringBuilder runtimeVersion;
-                uint hresult;
-#if DEBUG
-                // Just to make sure and exercise the code that doubles the size
-                // every time GetRequestedRuntimeInfo fails due to insufficient buffer size.
-                int bufferLength = 1;
-#else
-                int bufferLength = 11; // 11 is the length of a runtime version and null terminator v2.0.50727/0
```

Comment:
This means that previously it usually took only one P/Invoke to get the version. The new code always calls twice. I think it should be reasonable to make the first call with the estimate of 11 and only if it fails make another allocation based on the returned length.